index.html

```diff
@@ -6,170 +6,213 @@
-        <li><a href="#"><img src="./image/VectorGit.png" alt=""></a></li>
-        <li><a href="#"><img src="./image/Linkedin.svg" alt=""></a></li>
-    </ul>
-
-</div>
-<div class="buttonsee">
-    <button class="button1"> Get my resume </button>
-</div>
+    
+    <div class = "social-section">
+        <h4 class="connectmywork"> LET’S CONNECT</h4>
+    </div>
+            <ul class="social-icons-mywork">
+                <li><a href="#"><img src="./image/Vectortweet.png" alt=""></a></li>
+                <li><a href="#"><img src="./image/VectorM.jpg" alt=""></a></li>
+                <li><a href="#"><img src="./image/Vector5.png" alt=""></a></li>
+                <li><a href="#"><img src="./image/VectorGit.png" alt=""></a></li>
+                <li><a href="#"><img src="./image/Linkedin.svg" alt=""></a></li>
+            </ul>
+                <button class="button1 buttonsee"> Get my resume </button>
+    </div>
+    <div>
         <ul class="Program-skil">
-            <li  class="Programs-lang" href="#"><img  src="./image/Frame 3.png" alt=""></li>
-            <ul class="Program-skil">
-                <li  class="Programs" href="#"><img src="./image/Frame 9.png" alt=""></li>
-                  <li  class="Programs" href="#"><img src="./image/Frame 10.png" alt=""></li>
-                  <li  class="Programs" href="#"><img src="./image/Frame 11.png" alt=""></li>
-              </ul>
-            <li  class="Programs-lang" href="#"><img src="./image/Frame 01.png" alt=""></li>
-            <li  class="Programs-lang" href="#"><img src="./image/Frame 02.png" alt=""></li>
+                <li  class="Programs-lang" href="#"><img  src="./image/Frame 3.png" alt=""></li>
+            <ul class="Program-skil3">
+                <li  class="mobPrograms" href="#"><img src="./image/Frame 9.png" alt=""></li>
+                <li  class="mobPrograms" href="#"><img src="./image/Frame 10.png" alt=""></li>
+                <li  class="mobPrograms" href="#"><img src="./image/Frame 11.png" alt=""></li>
+
+                 <li  class="Programs" href="#"><img src="./image/js-vertical.png"   alt=""></li>
+                 <li  class="Programs" href="#"><img src="./image/html-vertical.png" alt=""></li>
+                 <li  class="Programs" href="#"><img src="./image/css-vertical.png" alt=""></li>
+            </ul>
+                    <li  class="Programs-lang" href="#"><img src="./image/Frame 01.png" alt=""></li>
+                    <li  class="Programs-lang" href="#"><img src="./image/Frame 02.png" alt=""></li>
         </ul>
-  </div>
+    </div>
+</div>
+  </section>
+  <section class="Workssection">
+    <form class="cardcontact" action="https://formspree.io/f/mwkzayzq" method="post" >
+        
+        <div class="background1">
+            <h2  class="contactme" >Contact me </h2>
+        <p class="sectioncontact"> If you have an application you are interested 
+            in developing, a feature that you need built or a project that 
```

Comment:
- There are still some things to improve in this section too, keep it up! 💯 

- Please, consider fixing the following:
  - [ ] The background image doesn't match the design, you're using the same image from the mobile version. 📱 
  - [ ] There's no `"Get in touch"` button in the desktop version, please add it! 💻 
  - [ ] You could use a `text-alignment: center` for your text and also add some padding to your input fields! 📜 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/201487537-679ed6b6-f4c6-4bf6-9341-a56cd7f55699.png) | ![image](https://user-images.githubusercontent.com/77069011/201487517-c5b317b9-93ec-4d2b-9cd6-c115cd43e223.png) |

Please, implement the requested changes! 🤓 🛠️